build.gradle

```diff
@@ -500,6 +500,12 @@ dependencyManagement {
             entry 'logback-classic'
             entry 'logback-core'
         }
+        dependencySet(group: 'com.fasterxml.jackson.core', version: '2.13.2') {
+            entry 'jackson-databind'
+            entry 'jackson-core'
+            entry 'jackson-annotations'
+            entry 'jackson-datatype-jsr310'
```

Comment:
this(jackson-datatype-jsr310) is already in line 355. So its duplicate.

Reply:
I missed the dependency - the duplicate has been removed.